### ISSUE TITLE
metrics graph

### ISSUE THREAD

**Issue by k3ybladewielder** [catboost/catboost#1873]
Im trying to plot the metrics graph and its not working, the plot is not showing

_model = CatBoostClassifier(iterations=100, custom_loss = ['AUC', 'Accuracy'])
model.fit(train_pool, eval_set = val_pool, verbose = False, plot = True)_
